CHANGELOG.asciidoc

```diff
@@ -256,6 +255,8 @@ https://github.com/elastic/beats/compare/v6.0.0-beta2...master[Check the HEAD di
 - Making the MongoDB module GA. {pull}6554[6554]
 - Allow to disable labels `dedot` in Docker module, in favor of a safe way to keep dots. {pull}6490[6490]
 - Add experimental module to collect metrics from munin nodes. {pull}6517[6517]
+- Add connections metricset to RabbitMQ module {pull}6548[6548]
```

Comment:
This line should be removed for this PR.

Reply:
Sure, I can remove it, but currently this line is in the wrong section - Bugfixes, while it should be in Added.

Reply:
I left the line in the change log, let me know if I should revert it.

Reply:
I'm just realising now that this log line has been moved and not added.

+1 on moving it to the right section.